addons/web/static/src/core/utils/draggable_hook_builder.js

```diff
@@ -662,8 +662,13 @@ export function makeDraggableHook(hookParams) {
                 // https://bugzilla.mozilla.org/show_bug.cgi?id=1352061
                 // https://bugzilla.mozilla.org/show_bug.cgi?id=339293
                 safePrevent(ev);
-                if (document.activeElement && !document.activeElement.contains(ev.target)) {
-                    document.activeElement.blur();
+                const eventDocument = ev.target.ownerDocument || ("activeElement" in ev.target && ev.target);
+                if (
+                    eventDocument &&
+                    eventDocument.activeElement &&
+                    !eventDocument.activeElement.contains(ev.target)
```

Comment:
The issue is that if the activeElement and the target are the same then contains will return true. [Source](https://developer.mozilla.org/fr/docs/Web/API/Node/contains)
Is that expected here?

Reply:
Updated the commit message as it seems I was pointing at the wrong commit...

Seems this was added in the specific case where you write in a field inside a dialog, and then drag the dialog by its header. So here the goal seems to be "if what I'm clicking isn't a descendant of the activeElement, it may be relevant to blur the activeElement in case it relies on that for saving".

So it should be fine that it does not fire if the target is a document, it just needs to fire when the target is strictly a parent.